docs/production/install-existing-server.md

```diff
@@ -2,9 +2,9 @@
 :orphan:
 ```
 
-# Production installation on existing server
+# Production installation on an existing server
```

Comment:
@gnprice said on CZO:
> The one adding "an" in a heading isn't exactly necessary, because it's normal for headings to omit articles, but the rest are all definitely good.